modules/nw-multus-tap-setting-boolean.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/secondary_networks/creating-secondary-nwt-other-cni.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-multus-tap-setting-boolean.adoc_{context}"]
+= Setting SELinux boolean for the TAP CNI plugin
+
+[role="_abstract"]
+To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using the Machine Config Operator (MCO).
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a new YAML file named, such as `setsebool-container-use-devices.yaml`, with the following details:
++
```

Comment:
```suggestion
.Example `setsebool-container-use-devices.yaml`
```
i know, i know, but i hear that a blocktitle solution is coming soon

Reply:
:heavy_check_mark: